src/serlio/modifiers/PRTModifierAction.cpp

```diff
@@ -600,6 +601,89 @@ MStatus PRTModifierAction::updateUI(const MObject& node) {
 	return MStatus::kSuccess;
 }
 
+MStatus PRTModifierAction::updateDynamicEnums() {
+	for (auto& e : mEnums) {
+		if (e.mValuesAttr.length() > 0) {

```

Comment:
Minor: one of these cases where the inverted statement (`if (val == 0) continue`) could avoid a large block of indentation

Reply:
fixed.